### ISSUE TITLE
Add city council district and community districts to schemas

### ISSUE THREAD

**Issue by TangoYankee** [NYCPlanning/ae-zoning-api#221]
# Description
Add city council district and community districts to schemas
- [ERD](https://github.com/NYCPlanning/ae-data-flow/pull/51/files#diff-88e04917ca937a375970c6fef6c1bd608a2843cfeec47962539a746d00c89fc8)
  - **NOTE**: The borough table is shown for reference in the ERD. It is already implemented in the zoning api and this existing table should be used to link the community_district
- [city_council_district table definition](https://github.com/NYCPlanning/ae-data-flow/pull/51/files#diff-b5c9eaa22770b61622a19cdbd4449fa1c5c61bc04f37eb22c862f1870369cfa1R216)
- [community_district table defintion](https://github.com/NYCPlanning/ae-data-flow/pull/51/files#diff-b5c9eaa22770b61622a19cdbd4449fa1c5c61bc04f37eb22c862f1870369cfa1R229)

## Acceptance criteria
- [x] A `city_council_district` schema exists and mirrors the columns documented in the ERD/reference SQL table
- [x] A `community_district` schema exists and mirrors the columns documented in the ERD/reference SQL table
- [x] The database columns use snake_case and the javascript variables use camelCase

**Note** Do not add the spatial fields to the zod schemas. They are not part of the standard response.

# Related tickets
blocked by: nycplanning/ae-data-flow#48
